client/lib/plans/constants.js

```diff
@@ -120,6 +123,7 @@ export const featuresList = {
 
 	[ FEATURE_UNLIMITED_STORAGE ]: {
 		getTitle: () => i18n.translate( 'Unlimited Storage' ),
+		getStoreSlug: () => 'unlimited_space',
```

Comment:
I think they may be allowed to renew if customers have grandfathered upgrades. For example some folks are on the free plan, but with a no-ads upgrade.